workspace-manager-clienttests/build.gradle

```diff
@@ -0,0 +1,84 @@
+import org.gradle.api.tasks.JavaExec
+
+plugins {
+    id 'java'
+    // Apply the application plugin to add support for building a CLI application in Java.
+    id 'application'
+    // Terra Test Runner Plugin
+    id 'bio.terra.test-runner-plugin' version '0.0.1-SNAPSHOT'
+}
+
+repositories {
+    // Use JCenter for resolving dependencies.
+    jcenter()
+    maven {
+        url 'https://broadinstitute.jfrog.io/broadinstitute/libs-release-local/'
+    }
+    maven {
+        url 'https://broadinstitute.jfrog.io/broadinstitute/libs-snapshot-local/'
+    }
+}
+
+dependencies {
+    ext {
+        apacheMath = '3.6.1'
+        findbugsAnnotations = "3.0.1"
+        guava = "29.0-jre"
+        jackson = "2.10.2"
+        jersey = "2.30.1"
+        kubernetesClient = "8.0.2"
+        logback = "1.2.3"
+        slf4j = "1.7.25"
+        hamcrest = "2.1"
+        swaggerAnnotations = "2.1.5"
+
+        googleOauth2 = "0.20.0"
+
+        workspacemanagerClient = "0.6.0-SNAPSHOT"
+        datarepoClient = "1.0.155-SNAPSHOT"
+        testRunner = "0.0.1-SNAPSHOT"
+    }
+    implementation group: 'org.apache.commons', name: 'commons-math3', version: "${apacheMath}"
+    compileOnly "com.google.code.findbugs:annotations:${findbugsAnnotations}"
+    implementation group: 'com.google.guava', name: 'guava', version: "${guava}"
+    compile "com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:${jackson}"
+    compile "org.glassfish.jersey.inject:jersey-hk2:${jersey}"
+    compile "io.kubernetes:client-java:${kubernetesClient}"
+    compile "ch.qos.logback:logback-classic:${logback}"
+    compile "org.slf4j:slf4j-api:${slf4j}"
+    compile "org.hamcrest:hamcrest:${hamcrest}"
+
+    // Google dependencies
+    compile "com.google.auth:google-auth-library-oauth2-http:${googleOauth2}"
+
+    // Data Repo client library
+    implementation "bio.terra:datarepo-client:${datarepoClient}"
+    compile "io.swagger.core.v3:swagger-annotations:${swaggerAnnotations}"
+
+    // Workspace Manager Service client library
+    //implementation "bio.terra:workspace-manager-client:${workspacemanagerClient}"
+
+    // Workspace Manager client library
+    // Gradle project property "workspacemanagerclientjar" overrides the fetch from Maven
+    if (project.hasProperty("workspacemanagerclientjar")) {
+        implementation files(project.findProperty("workspacemanagerclientjar"))
+        compile "com.fasterxml.jackson.datatype:jackson-datatype-jsr310:${jackson}"
+        compile "org.glassfish.jersey.core:jersey-client:${jersey}"
+        compile "org.glassfish.jersey.media:jersey-media-json-jackson:${jersey}"
+        compile "org.glassfish.jersey.media:jersey-media-multipart:${jersey}"
+        compile "org.glassfish.jersey.inject:jersey-hk2:${jersey}"
+    } else {
+        implementation "bio.terra:workspace-manager-client:${workspacemanagerClient}"
+        compile "io.swagger.core.v3:swagger-annotations:${swaggerAnnotations}"
+    }
```

Comment:
Refer to Mariko's comments above:

_I think we need to add a step to build the client library from the current branch's code. Otherwise, it's testing new code with an old library. That will cause confusion when changes affected the client library._

Reply:
apologies for not being clear ... what is the purpose of the `if` clause here? How/when does someone specify `workspacemanagerclientjar`? Should that be explained in the readme?

Reply:
Un-resolved this comment...to make sure the open question is still visible.